exchanges/orderbook/orderbook_test.go

```diff
@@ -473,16 +396,16 @@ func TestProcessOrderbook(t *testing.T) {
 	wg.Wait()
 }
 
-func deployUnorderedSlice() Tranches {
-	ts := make([]Tranche, 1000)
+func deployUnorderedSlice() Levels {
```

Comment:
🚧 Sorry I didn't spot this one earlier
```suggestion
func levelsFixtureRandom() Levels {
```